azure-pipelines/azure-templates/docker-template.yml

```diff
@@ -0,0 +1,56 @@
+
+parameters:
+  - name: 'displayName'
+    type: string
+  - name: 'targetPath'
+    type: string
+  - name: 'artifact'
+    type: string
+  - name: 'tags'
+    type: string
+  - name: 'Dockerfile'
+    type: string
+  - name: 'repository'
+    type: string
+  - name: 'stage'
+    type: string
+
+stages:
+- stage: ${{ parameters.stage }}
+  displayName: ${{ parameters.displayName }}
+  dependsOn: Build_and_Test
+  jobs:
+  - job: BuildPushAzure
+    steps:
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'current'
+        artifactName: ${{ parameters.artifact }}
+        itemPattern: '**.jar'
+        targetPath: ${{ parameters.targetPath }}
+    - task: Docker@2
+      inputs:
+        containerRegistry:
+        repository: $(service)
+        command: 'buildAndPush'
+        Dockerfile: ${{ parameters.Dockerfile }}
+        tags: ${{ parameters.tags }}
+
+  - job: BuildPushDocker
+    steps:
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'current'
+        artifactName: ${{ parameters.artifact }}
+        itemPattern: '**.jar'
+        targetPath: './dockerize/'
+    - task: Docker@2
+      inputs:
+        containerRegistry:
+        repository: ${{ parameters.repository }}
+        command: 'buildAndPush'
+        Dockerfile: ${{ parameters.Dockerfile }}
+        tags: ${{ parameters.tags }}
+        
+  - template: ./discord-job-template.yml
```

Comment:
don't you want this job to run always (`condition`/`dependsOn`), and at the end of any pipeline (even the ones that don't use this template - or do you expect all your pipelines to use this template)?

Reply:
^ previous comment appears not addressed